test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj

```diff
@@ -52,6 +52,7 @@
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="6.0.0-preview.7.21378.6" />
     <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="3.1.2.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="3.0.29" />
```

Comment:
Was this on purpose?

Reply:
Yeah thanks for catching this. I think I had added this to try and make some of the unit tests work but didn't end up needing it. Needed it for the integration tests though